tools/third-party-dependencies/known-dependencies.txt

```diff
@@ -130,4 +130,5 @@ httpcore-nio-4.4.6.jar
 javassist-3.21.0-GA.jar
 nacos-client-2.0.4.jar
 reflections-0.9.11.jar
-snakeyaml-1.23.jar
\ No newline at end of file
+snakeyaml-1.23.jar
+spring-boot-starter-web-2.6.7.jar
```

Comment:
Right now, the example module will not be packaged into distribution, so we may don't need to add license of `spring-boot-starter-web-2.6.7`? Actually, I suggest we add example into our distribution. cc @xwm1992

Reply:
So ? You decide .

Reply:
> Right now, the example module will not be packaged into distribution, so we may don't need to add license of `spring-boot-starter-web-2.6.7`? Actually, I suggest we add example into our distribution. cc @xwm1992

not only for distribution, theses modules are all upload to the maven repository.

Reply:
> Currently, the check license script will only check the third-part libs in distribution. If we didn't add the example module in distribution, we can't add `spring-boot-starter-web-2.6.7.jar ` in this file, otherwise the license check will fail.

Ok, I get what you mean,  we either only upgrade the `spring-boot-starter-web-2.6.7.jar` version and not add this to the file `known-dependencies.txt`, or add the `example moudule` into the distribution ? I prefer the former, not add  `spring-boot-starter-web-2.6.7.jar` to the files, because the `example` don't need to be used in distributions. @misselvexu @ruanwenjun what's your opinion?

Reply:
Already removed temporarily .

Reply:
In this PR, we don't need to add `pring-boot-starter-web-2.6.7.jar` into `known-dependencies.txt `. We can add another PR to add example module into distribution.

Reply:
Before submit PR, you can discuss this in issue or mail, I am not sure if this is needed.